### PR TITLE
Make _ctror_var_reset and _configure_coverage static.

### PR DESCRIPTION
**Pull request by gezalore**
Another step towards #2958/#2140. Make the mentioned generated functions
static for modules (but not for classes).

### CODE REVIEW

File: src/V3EmitC.cpp [hunk 1]
```diff
@@ -1753,7 +1753,9 @@ class EmitCImp final : EmitCStmts {
     }
 
     void emitVarReset(AstVar* varp) {
-        AstNodeDType* dtypep = varp->dtypep()->skipRefp();
+        AstNodeDType* const dtypep = varp->dtypep()->skipRefp();
+        const string varName
```
Comment (wsnyder) [inline src/V3EmitC.cpp:1757]:
```suggestion
        const string varNameProtected
```

And likewise rename in other places.  This makes it easier to audit/understand what being printed is protected versus not.

**Review by wsnyder — COMMENTED** [review]
This is an improvement, but ideally there would be less going on in V3EmitC and not have a magic number of functions needing to be passed to it  (because V3Emit is too complicated, and because maybe there will be later optimizations in between V3CCtors and V3EmitC at some point).  That is have the CCTors code look at the number of functions made. If one, rename the single function to __vl_CoverInsert.  If more than one, make a new _vl_CoverInsert function that calls all the other generated functions.  Ditto _ctor_var_reset.

**Comment by gezalore** [comment]
Yep, that's clearly a better way, done that, thank you.

**Review by wsnyder — COMMENTED** [review]
Thanks for the cleanup. Feel free to squash merge when tests and you're happy.

**Pull request merged**